src/components/panel/panel.spec.js

```diff
@@ -788,7 +889,6 @@ describe('$mdPanel', function() {
         openPanel(config);
 
         var panelCss = document.querySelector(PANEL_EL).style;
-        expect(panelCss.position).toEqual('fixed');
```

Comment:
Can't access the value. It is now set in a css class and the style sheets aren't loaded in tests.